argo-cd-apps/overlays/development/kustomization.yaml

```diff
@@ -56,3 +56,8 @@ patches:
       kind: ApplicationSet
       version: v1alpha1
       name: image-controller
+  - path: development-overlay-patch.yaml
```

Comment:
Prom isn't deployed currently in dev. So we need to omit the o11y app from it. https://github.com/redhat-appstudio/infra-deployments/blob/main/argo-cd-apps/overlays/development/delete-applications.yaml

Reply:
do you mean it should be on delete *instead* of of this kustomization, or in addition to it? (I modified to the former)

Reply:
the change that you've made is ok.